solr/core/src/java/org/apache/solr/servlet/SolrDispatchFilter.java

```diff
@@ -133,6 +134,9 @@ public SolrDispatchFilter() {}
   @Override
   public void init(FilterConfig config) throws ServletException {
     try {
+      SLF4JBridgeHandler.removeHandlersForRootLogger();
+      SLF4JBridgeHandler.install();
```

Comment:
This logic should ideally be done as early as possible.  A ServletFilter's init method is not actually so early as we can do.  We could do CoreContainerProvider.init() which will start up CoreContainer _before_ the ServletFilter.  But we could _do earlier than that_ by **adding some XML based configuration in jetty.xml, which I think makes more sense**.  It could also mean we add the dependency JAR without formerly making it a dependency of solr-core, thus wouldn't leak to other/external things projects that might want to depend on solr-core.  This is a deployment matter so it's best done at the Jetty layer.

Reply:
Would that change be in this file? https://github.com/apache/solr/blob/457ca3668a8906277f8f638af9c740b57f2a2a20/solr/server/etc/jetty.xml#L8
Do you know what it would look like?

Reply:
A quick google search turned up [this](https://github.com/rapla/rapla/blob/master/src/test/etc/jetty.xml) immediately.